includes/models/model.llms.user.certificate.php

```diff
@@ -106,16 +106,19 @@ public function __construct( $model, $args = array() ) {
 	}
 
 	/**
-	 * Called immediately after creating / inserting a new post into the database
+	 * Maybe set this awarded certificate sequential id based on the parent's meta.
 	 *
 	 * @since [version]
 	 *
 	 * @return void
 	 */
-	protected function after_create() {
+	public function maybe_set_sequential_id() {
```

Comment:
I'm being nitpicky but I don't like the idea of a "maybe" setter... I know we use the "maybe_do_x" pattern quite a bit, but usually in a callback function for a hook/filter, not in a function that is part of the truly public API.

My suggestion here would be to change this to `update_sequential_id()` and add a return to the function so that if there's no parent, it returns `false` 

If there *is* a parent, it then get's the next sequential ID, sets it, and then returns it.

Does this seem stupid and nitpicky?

Reply:
No it's fine. ❤️